lottie/src/main/java/com/airbnb/lottie/BaseLayer.java

```diff
@@ -325,7 +326,17 @@ private void buildParentLayerListIfNeeded() {
     }
   }
 
+  @Nullable @Override public String getName() {
```

Comment:
it is possible when parsing the json, the return value can be null

It is just to be clear to future contributors that a null value is possible